client/src/containers/AuthContainer.tsx

```diff
@@ -29,70 +31,70 @@ function TabPanel(props: TabPanelProps) {
       aria-labelledby={`full-width-tab with index: ${index}`}
       {...rest}
     >
-      {value === index && (
-        <Box>
-          <div>{children}</div>
-        </Box>
-      )}
+      {value === index && <Box sx={{ p: 3 }}>{children}</Box>}
     </div>
   );
 }
 
 function AuthContainer() {
-  const paperStyle = {
-    height: "100%",
-    borderRadius: "20px",
-  };
+  const theme = useTheme();
 
   const [tabIndex, setTabIndex] = useState(0);
 
-  const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
     setTabIndex(newValue);
   };
 
-  const matches = useMediaQuery("(min-width: 1080px)");
+  const matches = useMediaQuery(theme.breakpoints.down("md"));
 
   return (
     <div className="AuthContainer">
       <Grid
         container
-        justifyContent={matches ? "flex-end" : "center"}
-        style={{ padding: matches ? "20px" : 0 }}
+        component="main"
+        justifyContent="flex-end"
+        alignItems="center"
+        sx={{ height: "100vh", px: "40px" }}
       >
-        <Grid item md={7} lg={8} xl={9}>
+        <Grid item hidden={matches} xs={false} md={8}>
           <Box
-            style={{
+            sx={{
               textAlign: "center",
-              paddingTop: matches ? "100px" : "0px",
             }}
-            fontFamily="Verdana"
-            color="white"
           >
-            <h1 style={{ fontSize: matches ? "500%" : "200%" }}>UOFTBOARD</h1>
-            <h1>A place to connect</h1>
+            <Typography color="grey" fontFamily="Verdana" variant="h2">
```

Comment:
Change the font color, it is clashing with the background.

Reply:
I am aware but looking through all the available colours is hard, unless you know of a good colour to use

Reply:
Fixed